build/check.py

```diff
@@ -121,7 +121,8 @@ def get(*args):
 
   # Ignore missing goog.require since we assume the whole library is
   # already included.
-  opts = ['--jscomp_off=missingRequire', '--checks-only', '-O', 'SIMPLE']
+  opts = ['--jscomp_off=missingRequire', '--jscomp_off=lintChecks',
```

Comment:
Not off-hand, but I do note that all of those lint check failures are about goog.requires in demo/ and test/, which do not use goog.require in the first place.  I would strongly prefer not to disable useful checks on the library just to exclude them from test and demo app code.